.travis.yml

```diff
@@ -2,19 +2,24 @@ language: python
 dist: xenial  # for python3.7+ testing
 
 python:
-- 3.5
 - 3.6
```

Comment:
did you mean to drop the whole v 3.5? or just the extra dash?

Reply:
Drop all of v3.5, `loompy` doesn't support it so testing against it seems to be more trouble than it's worth.